bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/BookieServer.java

```diff
@@ -127,7 +127,7 @@ public void start() throws InterruptedException, IOException {
         if (!this.bookie.isRunning()) {
             exitCode = bookie.getExitCode();
             this.requestProcessor.close();
-            return;
+            throw new InterruptedException("fail fast, bookie startup is not successful");
```

Comment:
We would better define another Exception; the `InterruptedException` is for the `thread` case.

Reply:
Can we define it "BookieStartException", extends the BookieException ?

Reply:
Sounds good.

Reply:
have done.